index.js

```diff
@@ -12,7 +12,7 @@ module.exports = function (fastify, opts, next) {
     ? /^index\.(ts|js)$/im
     : /^index\.js$/im
   const scriptPattern = opts.includeTypeScript
-    ? /\.(ts|js)$/im
+    ? /((?<!\.d)\.ts|\.js)$/im // For .ts files, ignore .d.ts, see #65
```

Comment:
CI is failing for

> not ok Invalid regular expression: /((?<!\.d)\.ts|\.js)$/: Invalid group

Could you change the regexp to avoid the Negative lookbehind?

Reply:
Yeah I did that on my fork, but the commit did not make it here, that's weird..